bundles/org.openhab.ui.dashboard/web/js/home-designer.js

```diff
@@ -302,4 +460,9 @@ var vue = {
     }
 };
 
-var vm = new Vue(vue);
\ No newline at end of file
+var vm = new Vue(vue);
+
+// TODO: Use a Promise instead
+$(document).ready(() => {
+    vm.setLanguage();
```

Comment:
This doesn't work yet as the REST response comes asynchronously (as it should). I'm gonna use Promise instead of waiting for document to be loaded.
Maybe I'll try not to use jQuery at all, since this is the only place I'm using `$`.